tests/test_harvest_pagination.py

```diff
@@ -58,8 +59,6 @@ def setUpClass(cls):
             if cls._master[stream]['test'] or stream == 'projects':
                 _, record_count = get_stream_counts(stream)
                 logging.info("{} has {} records".format(stream, record_count))
```

Comment:
Remove logging and use tap_tester LOGGER instead

Reply:
Replaced `logging` with `LOGGER`.